### DISCUSSION TITLE
What can I do if the sitemap URL is rejected in Google Search Console?

### DISCUSSION THREAD

**Discussion by ApertureBMX** [datenstrom/community#526] [Ask a question] [answered]
Hi. I'm using the sitemap extension to generate a sitemap for my website. However, Google Search Console doesn't seem to accept the page:sitemap.xml URL.

Is there a way to generate the sitemap at an alternative, simpler URL such as sitemap/sitemap.xml?

**Comment by ApertureBMX** [comment]
Hmm, now Google seems to have fetched the sitemap. It'd still be nice if the extension would generate a simpler URL, but I guess it's working.

**Comment by markseuffert** [comment, marked as answer]
The sitemap is available as `http://website/sitemap/` and `http://website/sitemap/page:sitemap.xml`, this is supported out of the box. For a different URL, such as `http://website/sitemap.xml`, you probably have to make a new extension or upload a `sitemap.xml` file to the root folder.
